.env

```diff
@@ -4,17 +4,20 @@ COMPOSE_PROJECT_NAME=artemis
 DB_VERSION=12
 GUI_ENABLED=true
 SYSTEM_VERSION=latest
+HISTORIC=false
```

Comment:
make sure you document all the updated .env variables in the respective wiki page